maps/GWDM/2.0/HDRUK/3.0.0/translation.jsonata

```diff
@@ -26,14 +26,14 @@ $joinTypicalAgeRange := function($min, $max){(
     $minStr ? $maxStr ? $join([$minStr, "-", $maxStr]) : null : null
 )};
 
-$getDatasetLinkage := function(){(
+$getDatasetLinkage := function() {(
     $datasetLinkage := {
-      "isDerivedFrom":$to_str(input.enrichmentAndLinkage.derivedFrom),
-      "isPartOf":$to_str(input.enrichmentAndLinkage.isPartOf),
-      "isMemberOf": $to_str(input.enrichmentAndLinkage.similarToDatasets),
-      "linkedDatasets":$to_str(input.enrichmentAndLinkage.linkableDatasets)
+        "isDerivedFrom": $join($map(input.enrichmentAndLinkage.derivedFrom, function($v) { $v.url ? $v.url : $v.title }), ", "),
```

Comment:
We usually use `;,;` as a delimiter in the DB, to distinguish it from plain commas in strings.

Reply:
Fixed!